package.json

```diff
@@ -949,6 +949,10 @@
                                     "type": "string"
                                 }
                             },
+                            "platform": {
+                                "type": "string",
+                                "description": "%vscode-docker.tasks.docker-build.dockerBuild.platform%"
```

Comment:
Is there a definitive list I could use to supply enum values? That would be handy.

Reply:
@bwateratmsft it looks like the platforms/architectures are based on an underlying Go supported platform list, with some normalization of inputs done by Docker:
https://github.com/containerd/containerd/blob/main/platforms/platforms.go
https://go.dev/doc/install/source#environment